pkg/kubelet/cm/cpumanager/options_test.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
I would just add these to the existing `policy_static_test.go` file.

Reply:
ack